packages/bpk-component-modal/src/BpkModalDialog-test.js

```diff
@@ -70,6 +70,45 @@ describe('BpkModalDialog', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('should render correctly with wide prop', () => {
```

Comment:
this branch of the code was not covered so I've added the test to bring code coverage to 100%